controller_manager/controller_manager/__init__.py

```diff
@@ -12,8 +12,101 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import launch_utils
+from controller_manager_msgs.srv import ConfigureController, \
+    ListControllers, ListControllerTypes, ListHardwareInterfaces, \
+    LoadController, ReloadControllerLibraries, SwitchController, UnloadController
+
+import rclpy
+
+
+def service_caller(node, service_name, service_type, request):
+    cli = node.create_client(service_type, service_name)
+
+    if not cli.service_is_ready():
+        node.get_logger().debug('waiting for service {} to become available...'
+                                .format(service_name))
+        if not cli.wait_for_service(2.0):
+            raise RuntimeError('Could not contact service {}'.format(service_name))
+
+    node.get_logger().debug('requester: making request: %r\n' % request)
+    future = cli.call_async(request)
+    rclpy.spin_until_future_complete(node, future)
+    if future.result() is not None:
+        return future.result()
+    else:
+        raise RuntimeError('Exception while calling service: %r' % future.exception())
+
+
+def configure_controller(node, controller_manager_name, controller_name):
```

Comment:
finally figured out what puzzled me with the balance of this PR, you didn't remove these calls from `ros2controlcli/ros2controlcli/api/__init__.py` but simply duplicated 'em here. Please remove them from there, if you want to keep """"API compatibility"""" (I'm fairly sure nobody uses this) you can still export these calls from that `__init__.py` too